src/renderer/components/modals/DepositWithdrawModal.vue

```diff
@@ -68,47 +67,6 @@ export default {
       this.amount = this.isDeposit ?
         this.holding.balance.toString() :
         this.holding.contractBalance.toString();
-
-      this.cleanAmount();
```

Comment:
No longer needed. When clicking the 'max'  button it will also fires the inputs 'blur' event, which then calls the 'cleanAmount' formatting method.